diff --git a/BUILD b/BUILD
index d7c731bf6..315cdeca0 100644
--- a/BUILD
+++ b/BUILD
@@ -65,6 +65,9 @@ boringssl_copts = select({
     # This is needed on Linux systems (at least) to get rwlock in pthread.
     "-D_XOPEN_SOURCE=700",
 
+    # Disable assembly code for testing cross-compilation
+    "-DOPENSSL_NO_ASM",
+
     # This list of warnings should match those in the top-level CMakeLists.txt.
     "-Wall",
     "-Werror",
@@ -88,6 +91,7 @@ boringssl_copts = select({
     ":windows_x86_64": [
         "-DWIN32_LEAN_AND_MEAN",
         "-DOPENSSL_NO_ASM",
+        "-DBORINGSSL_IMPLEMENTATION",
     ],
     "//conditions:default": ["-DOPENSSL_NO_ASM"],
 })
@@ -141,6 +145,7 @@ cc_library(
         ":windows_x86_64": ["-defaultlib:advapi32.lib"],
         "//conditions:default": ["-lpthread"],
     }),
+    linkstatic = True,
     visibility = ["//visibility:public"],
 )
 
@@ -150,6 +155,7 @@ cc_library(
     hdrs = ssl_headers,
     copts = boringssl_copts_cxx,
     includes = ["src/include"],
+    linkstatic = True,
     visibility = ["//visibility:public"],
     deps = [
         ":crypto",
